apps/test/unit/templates/VersionHistoryTest.jsx

```diff
@@ -210,11 +215,14 @@ describe('VersionHistory', () => {
       wrapper.find('.btn-danger').simulate('click');
 
       // Expect confirmation to show
-      expect(
```

Comment:
I don't think this `expect` was verifying anything without a `to` clause?  It's a little weird that we use both `expect` and `assert` in this file but I didn't try to address this.

Reply:
you're right about `expect` not doing anything there